tests/components/logbook/test_init.py

```diff
@@ -378,13 +386,15 @@ def test_include_events_entity(self):
             entries[1], pointB, "blu", domain="sensor", entity_id=entity_id2
         )
 
-    def test_include_events_domain(self):
-        """Test if events are filtered if domain is included in config."""
+    def test_include_events_domain_glob(self):
```

Comment:
We should not alter the behavior of existing tests beyond that is needed to make the function not reject argument changes, it is better to add new ones so we prove the old tests are still working without changes.

Reply:
Reverted changes to tests where I added glob filters and added separate ones specifically for that.